apps/docs/src/content/docs/reference/default.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Default task
+description: the task function
+---
+
+import { LinkCard, Tabs, TabItem, Aside } from '@astrojs/starlight/components';
```

Comment:
Should this be embedded in a code block?

Reply:
Nope this is an actual import of some components

Reply:
Yes sorry, I added this comment when I was looking at the markdown view from GitHub and I switched to the preview only afterward and forgot to delete ^^'